### PR TITLE
Fix storybook and add camelCase feature

### PR DESCRIPTION
**Pull request by HarryPerez**
## Summary
Storybook

#### Features:
* Fixed Storybook 
* Moved Storybook to execute it only at bootstrap level (**It nos not generated with the project anymore**)
* Added camelCase feature in order to use styles **`className={styles.lobrunoNovimo}`** like:
 ```
 .lobruno-novimo {
  width: 100px;
}
```

### CODE REVIEW

File: generators/app/.storybook/.babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@babel/preset-react", "@babel/preset-env"],
+  "plugins": ["@babel/plugin-proposal-class-properties"]
+}
```
Comment (FrankIglesias) [inline generators/app/.storybook/.babelrc]: add EOL

File: generators/app/.storybook/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (FrankIglesias) [inline generators/app/.storybook/webpack.config.js]: remove this line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: generators/app/.storybook/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const path = require('path');
+
+const addCamelCaseToCSSModulesByHarry = (config) => {
```
Comment (pabloccid) [inline generators/app/.storybook/webpack.config.js]: Rename this function 👀

Comment (pabloccid) [reply generators/app/.storybook/webpack.config.js]: Remove parentheses

File: generators/app/.storybook/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  const cssRule = config.module.rules.find(rule => String(rule.test) === `${path.sep}\\.module\\.(scss|sass)$${path.sep}`);
+  cssRule.use.forEach(attribute => {
+    attribute.options && (attribute.options.camelCase = true);
+  })
+  return config;
+};
+
+module.exports = baseConfig => addCamelCaseToCSSModulesByHarry(baseConfig);
```
Comment (pabloccid) [inline generators/app/.storybook/webpack.config.js]:
```suggestion
module.exports = addCamelCaseToCSSModulesByHarry;
```

**Review by pabloccid — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Review by pabloccid — APPROVED** [review]

**Pull request merged**